dev/ProgressBar/ProgressBar_themeresources.xaml

```diff
@@ -1,288 +1,43 @@
 ﻿<!-- Copyright (c) Microsoft Corporation. All Rights Reserved. -->
 <ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
-    xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <x:Double x:Key="ProgressBarIndicatorPauseOpacity">0.6</x:Double>
-            <x:Double x:Key="ProgressBarThemeMinHeight">4</x:Double>
             <Thickness x:Key="ProgressBarBorderThemeThickness">0</Thickness>
-            <Color x:Key="SystemControlErrorBackgroundColor">#33FFFFFF</Color>
```

Comment:
This color is odd. SystemControlErrorBackgroundColor is defined for light and dark theme, not for HC. And it's not in generic.xaml so may be a breaking change if removed.

Reply:
it was used for color animations that we no longer use it in. the color animations were pretty wonky. and yes, there were no HC equivalent. so, definitely not a proper way to do it.

Reply:
so I'm confident we don't need this one any longer.